Sources/Networking/HTTPResponse.swift

```diff
@@ -59,22 +59,24 @@ extension HTTPResponse {
 // MARK: -
 
 /// The response content of a failed request.
-struct ErrorResponse {
+struct ErrorResponse: Equatable {
 
-    let code: BackendErrorCode
-    let message: String?
-    let attributeErrors: [String: String]
+    var code: BackendErrorCode
+    var message: String?
+    var attributeErrors: [String: String] = [:]
 
 }
 
 extension ErrorResponse {
 
     /// Converts this `ErrorResponse` into an `ErrorCode` backed by the corresponding `BackendErrorCode`.
-    func asBackendError(with statusCode: HTTPStatusCode) -> Error {
-        var userInfo: [NSError.UserInfoKey: Any] = [
-            ErrorDetails.finishableKey: !statusCode.isServerError,
-            Backend.RCSuccessfullySyncedKey: statusCode.isSuccessfullySynced
-        ]
```

Comment:
These are gone in favor of the properties in `NetworkError` that rely on the `HTTPStatusCode` implementation as well.